app/models/flood_risk_engine/enrollment_exemption.rb

```diff
@@ -22,6 +22,13 @@ class EnrollmentExemption < ActiveRecord::Base
       operator_failings: 1
     }
 
+    enum assistance_mode: {
+      unassisted: 0,
+      fully: 1,
+      partial: 2,
+      admin_intervention: 3
+    }
+
```

Comment:
Just wondered if the `fully` and `partial` enum values were e.g. `fully_assisted` and `partially_assisted` then calling `enrollment_exemption.fully_assisted!` would have more meaning than `enrollment_exemption.fully!`